README.md

```diff
@@ -66,3 +66,4 @@ The API endpoint should cover the following steps:
     - When the email delivery fails permanently.
       - Set the status of the Email object to "error"
       - Set the Httpstatuscode "400" to the response
+1. If false, the Email object should be ignored
```

Comment:
and it will stay there forever in the project? or is there a point where you clean up all these unneeded objects? @ahalberkamp

Reply:
not  now, but we have to think about it. maybe we can talk about it tomorrow...

Reply:
@ahalberkamp Maybe create an issue for it so it's not forgotten about :)?

Reply:
https://github.com/commercetools/commercetools-email-retry-processor/issues/2